docs/integrations/webhooks/overview.mdx

```diff
@@ -111,10 +111,20 @@ To replay webhook messages:
 
 You can find a guide on how to use webhooks to sync your data to your database [here](/docs/integrations/webhooks/sync-data).
 
-## Protect your webhooks from abuse
+## Webhook security
 
 To ensure that the API route receiving the webhook can only be hit by your app, there are a few protections you can put in place:
 
-- **Verify the request signature**: Svix webhook requests are [signed](https://www.wikiwand.com/en/Digital_signature) and can be verified to ensure the request is not malicious. To learn more, check out Svix's guide on [how to verify webhooks with the svix libraries](https://docs.svix.com/receiving/verifying-payloads/how) or [how to verify webooks manually](https://docs.svix.com/receiving/verifying-payloads/how-manual).
+- **Verify the request signature**: Svix webhook requests are [signed](https://www.wikiwand.com/en/Digital_signature) and can be verified to ensure the request is not malicious. To learn more, check out Svix's guide on [how to verify webhooks with the svix libraries](https://docs.svix.com/receiving/verifying-payloads/how) or [how to verify webhooks manually](https://docs.svix.com/receiving/verifying-payloads/how-manual).
 
 - **Only accept requests coming from [Svix's webhook IPs](https://docs.svix.com/webhook-ips.json)**:  To further prevent attackers from flooding your servers or wasting your compute, you can ensure that your webhook-receiving api routes only accept requests coming from [Svix's webhook IPs](https://docs.svix.com/webhook-ips.json), rejecting all other requests.
+
+## Development and Testing
+
+To begin developing webhooks, start by discovering all types of event payloads. For this, you need an HTTPS endpoint. Beeceptor provides free HTTPS endpoints for payload discovery and routing to a service running locally on your development box. This allows you to test locally and iterate quickly without regular deployment.
+Here is how you can get started:
+
+1. Create a free endpoint on Beeceptor: Visit [Beeceptor](https://beeceptor.com/webhook-integration/?utm_source=social&utm_campaign=clerk-docs) to create a free endpoint.
+2. Copy the endpoint URL and configure webhooks: Copy the endpoint URL and enter it in the webhooks configuration screen.
+3. Trigger events and discover payloads: Trigger some events and discover the incoming payloads on Beeceptor. These are shown as they happen.
+4. Route requests to a localhost port like 3000 or 8080: Use the Beeceptor CLI to route requests to a localhost port for local testing.
```

Comment:
Friend, we [declined this addition before](https://github.com/clerk/clerk-docs/pull/693) and are going to stick with our original decision of declining!

```suggestion
```